backend/app/api/endpoints/resource_library.py

```
@@ -23,15 +25,53 @@
 router = APIRouter()
+PUBLIC_TEAM_ICON_ASSET_TYPE = "public_team_icon"
 
 
 def _parse_tags(tags: str | None) -> list[str]:
     return [item.strip() for item in (tags or "").split(",") if item.strip()]
 
 
+@router.get("/assets/team-icons/{asset_id}", include_in_schema=False)
+def get_public_team_icon(
+    asset_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.16.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)